pkgs/development/python-modules/mididings/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, stdenv
+, python
+, fetchFromGitHub
+, pkgs
+, libjack2
+, alsaLib
+, pkgconfig
+, glib
+, boost
+, python36Packages
+}:
+
+buildPythonPackage rec {
+  pname = "mididings";
+  version = "unstable-2015-11-17";
+
+
+  patches = [ ./0001-add-boost-suffix.patch ];
+
+  LIBRARY_PATH="${(pkgs.boost.override {enablePython = true; inherit python; })}/lib";
+  
+  nativeBuildInputs = [ 
+	pkgconfig 
+	];
+
+  buildInputs = with pkgs; [
+      (boost.override {
+        enablePython = true;
+        inherit python;
+      })
+  	 glib
+         libjack2
+	 alsaLib
+    	pkgconfig 
+    ];
+
+  runtimeDeps = [
+    python36Packages.decorator
+  ];
+   propagatedBuildInputs = [ python36Packages.pytest 
+ 		   python36Packages.decorator 
+		   python36Packages.pyliblo 
+                   python36Packages.dbus-python
+                   python36Packages.pyinotify 
+                   python36Packages.tkinter  ];
+
+
+   checkInputs = [ python36Packages.pytest 
+ 		   python36Packages.decorator 
+		   python36Packages.pyliblo 
+                   python36Packages.dbus-python
+                   python36Packages.pyinotify 
+                   python36Packages.tkinter  ];
```

Comment:
You only need to add additional packages here. No need to duplicate `propagatedBuildInputs`.
Pytest should be here but not in `propagatedBuildInputs`.